### PR TITLE
Initial setup to make Vokuro  Bootstrap 4.1.1 ready

### PR DESCRIPTION
**Pull request by MoHasanJ**
Initial setup to make Vokuro Bootstrap 4.1.1 ready, 
Related libraries added including open-iconic-fonts, jquery-3.3.1. Added common footer.volt, updated views/index/index.volt for "User Panel Button". 

Essentially, now home page of Vokuro is completely changed to BS4.1.1, including footer, menu etc. rest of the changes coming in upcoming commits.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@
 composer.lock
 vendor/
 app/config/config.dev.php
+/nbproject/
```
Comment (sergeyklay) [inline .gitignore]: Please do not use this ignore file to define platform specific files.

For these purposes create a global `.gitignore` file, which is a list of rules for ignoring files in every Git repository on your computer.

See: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

File: app/views/partials/footer.volt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <div class="row">
+            <div class="col-md-4">
+                Made with love by the Phalcon Team
+            </div>
+            <div class="col-md-4">
+                {{ link_to("privacy", "Privacy Policy") }}
+                {{ link_to("terms", "Terms") }}
+            </div>
+            <div class="col-md-4">
+                © {{ date("Y") }} Phalcon Team.
```
Comment (sergeyklay) [inline app/views/partials/footer.volt]:
```diff
- © {{ date("Y") }} Phalcon Team.
+ &copy; {{ date("Y") }} Phalcon Team.
```

**Comment by MoHasanJ** [comment]
@sergeyklay , I have made changes as you advised. Let me know if anything is required.

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**